dm/worker/log.go

```diff
@@ -505,3 +527,36 @@ func CloneTaskLog(log *pb.TaskLog) *pb.TaskLog {
 func whetherNil(handler interface{}) bool {
 	return handler == nil || reflect.ValueOf(handler).IsNil()
 }
+
+// clearByPrefix clears all keys with the specified prefix.
+func clearByPrefix(txn *leveldb.Transaction, prefix []byte) error {
+	if txn == nil {
+		return errors.Trace(ErrInValidHandler)
+	}
+
+	var err error
+	iter := txn.NewIterator(util.BytesPrefix(prefix), nil)
+	batch := new(leveldb.Batch)
+	for iter.Next() {
+		batch.Delete(iter.Key())
+		if batch.Len() >= GCBatchSize {
+			err = txn.Write(batch, nil)
+			if err != nil {
+				iter.Release()
+				return errors.Annotatef(err, "delete kv with prefix % X until % X", prefix, iter.Key())
+			}
+			log.Infof("[worker log] delete kv with prefix % X until % X", prefix, iter.Key())
+			batch.Reset()
+		}
+	}
+	iter.Release()
+	err = iter.Error()
+	if err != nil {
+		return errors.Annotatef(err, "iterate kv with prefix % X", prefix)
```

Comment:
as log above, add range information?

Reply:
we can't know the range here, we are clearing all keys with the specified _prefix_, but we don't know the start-key and end-key.